### PR TITLE
Shapoval commit

### PR DESCRIPTION
**Pull request by Shapovall**

### CODE REVIEW

**Review by StayTeeVone — APPROVED** [review]
bad!

File: server.js [hunk 1]
```diff
@@ -1 +1,2 @@
 console.log("hello");
+console.log("OK")
```
Comment (buba3230) [inline server.js:2]: PLZ dont use capital case!!! this is the bad practice....

**Review by buba3230 — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (buba3230) [inline server.js:2]: asdasdasd

**Review by buba3230 — CHANGES_REQUESTED** [review]

**Review by buba3230 — DISMISSED** [review]

File: server.js (hunk 1 above)
Comment (buba3230) [inline server.js:2]: this is not OK

**Review by buba3230 — CHANGES_REQUESTED** [review]

**Pull request merged**